CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
```

Comment:
It should remain for the future changes:)

Reply:
Ok, I will add back the `Unreleased` header, as well as empty sections for the next unreleased version.